### DISCUSSION TITLE
Custom auth providers

### DISCUSSION THREAD

**Discussion by RagnarTheEverchosen** [supabase/supabase#29859] [Questions]
Hey, I would like to ask if there are any plans to add OpenID 2.0 as a provider. 

To be specific I mean the [Steam OpenID](https://partner.steamgames.com/doc/features/auth#website)/[OpenID](https://openid.net/). 

I know there has been several similar post on this topic  as Steam seems to be a highly demanded provider but up to know I haven't seen a concreate answer if this might come in the future or not.

**Comment by Revadike** [comment]
[I attempted a PR for Steam provider support](https://github.com/supabase/auth/pull/1919)
However, I think the main problem is Supabase's reliability on user emails, so I am not sure if this will (fully) work.
